### PR TITLE
[Windows] Run SourceKit-LSP tests from build.ps1

### PR DESCRIPTION
**Pull request by ahoppen**
Building and running the tests currently takes 2:10min locally on my machine, but quite a few tests are currently disabled on Windows.

### CODE REVIEW

**Comment by ahoppen** [comment]
https://github.com/swiftlang/sourcekit-lsp/pull/1772

@swift-ci Please smoke test

File: utils/build.ps1 [hunk 1]
```diff
@@ -1223,7 +1223,7 @@ function Build-SPMProject {
     [string] $Src,
     [string] $Bin,
     [hashtable] $Arch,
-    [switch] $Test = $false,
+    [string] $Test = "no", # options: no (swift build), yes (swift test), parallel (swift test --parallel)
```
Comment (compnerd) [inline utils/build.ps1:1226]: Please create an enumerator for this, don't use a string typed value.

Comment (ahoppen) [reply utils/build.ps1]: Good suggestion. Updated.

**Comment by ahoppen** [comment]
https://github.com/swiftlang/sourcekit-lsp/pull/1772

@swift-ci Please smoke test

**Review by bnbarham — APPROVED** [review]

**Pull request merged**